pkg/compose/build_bake.go

```diff
@@ -335,6 +335,14 @@ func (s *composeService) doBuildBake(ctx context.Context, project *types.Project
 	if err != nil {
 		return nil, err
 	}
+	endpoint, cleanup, err := s.propagateDockerEndpoint()
+	if err != nil {
+		return nil, err
+	}
+	cmd.Env = append(cmd.Env, endpoint...)
+	defer cleanup()
+
+	fmt.Println(strings.Join(cmd.Env, "\n"))
```

Comment:
Not sure we want to print the env variables of the command by default, WDYT?
```suggestion
	logrus.Debugf((strings.Join(cmd.Env, "\n"))
```